packages/devtools/lib/src/ui/analytics.dart

```diff
@@ -137,14 +145,16 @@ void screen(
   GTag.event(
       screenName,
       GtagEventDevTools(
-          event_category: screenViewEvent,
-          value: value,
-          user_app: userAppType,
-          user_build: userBuildType,
-          user_platform: userPlatformType,
-          devtools_platform: devtoolsPlatformType,
-          devtools_chrome: devtoolsChrome,
-          devtools_version: devtoolsVersion));
+        event_category: screenViewEvent,
+        value: value,
+        user_app: userAppType,
+        user_build: userBuildType,
+        user_platform: userPlatformType,
+        devtools_platform: devtoolsPlatformType,
+        devtools_chrome: devtoolsChrome,
+        devtools_version: devtoolsVersion,
+        ide_launched: ideLaunched,
+      ));
```

Comment:
nit: for flutter style add a trailing comma between `))` as well.

Reply:
Go it - done.